Client/Ecosia/UI/NTP/Cells/NTPLogoCell.swift

```diff
@@ -28,6 +28,9 @@ final class NTPLogoCell: UICollectionViewCell, ReusableCell, NotificationThemeab
         logo.translatesAutoresizingMaskIntoConstraints = false
         logo.clipsToBounds = true
         logo.contentMode = .scaleAspectFit
+        logo.isAccessibilityElement = true
+        logo.accessibilityIdentifier = "ecosia-logo"
```

Comment:
I believe we can utilise the centralized `public struct AccessibilityIdentifiers` as the source of truth for those IDs

Reply:
Nice, will do! I will also use an Ecosia sub struct on there like in [the other PR](https://github.com/ecosia/ios-browser/pull/441) to split from FF code.